### PR TITLE
Isc dhcpd improvements

### PR DESCRIPTION
**Pull request by l2isbad**
human readable names for pools

new pools format in the module configuration file

```yaml
leases:
  pools:
    wifi: '192.168.2.0/24'
    office: '192.168.3.0/24'
    dmz: '192.168.4.0/24'
    192.168.5.0/24: '192.168.5.0/24'
```

### CODE REVIEW

**Comment by l2isbad** [comment]
@ktsaou please do [compatibility test](https://github.com/firehol/netdata/pull/1448) (i can't reproduce it on my py2/3 vm's)

File: conf.d/python.d/isc_dhcpd.conf [hunk 1]
```diff
@@ -75,4 +78,6 @@
 #
 #leases:
 #  leases_path : '/var/lib/dhcp/dhcpd.leases'
-#  pools       : '192.168.3.0/24 192.168.4.0/24 192.168.5.0/24'
+#    pools:
```
Comment (ktsaou) [inline conf.d/python.d/isc_dhcpd.conf]: I think the spacing here is wrong. `pools` should be aligned with `leases_path`.
Yes, it commented, but uncommenting it and running it gives error.

**Comment by ktsaou** [comment]
Tried it on my dhcp servers and it works.
I am not sure what to check.

**Comment by ktsaou** [comment]
The ones I tried are python v2.

**Comment by l2isbad** [comment]
> I am not sure what to check.

Nothing special. If it works we are ok.
I asked you because there was a bug I could not reproduce ([your PR](https://github.com/firehol/netdata/pull/1448/commits/7cd999adc1314a2d5c3745b94444d3941fe8ee8c)).
I also used py2.

**Comment by l2isbad** [comment]
ready then

**Comment by ktsaou** [comment]
merged! thanks!

**Pull request merged**